manifests/dev/virt-handler.yaml.in

```diff
@@ -12,7 +12,7 @@ spec:
       labels:
         kubevirt.io: virt-handler
     spec:
-      serviceAccountName: kubevirt-infra
+      serviceAccountName: kubevirt-privileged
```

Comment:
If I am right and libvirt does not need a privileged service account, could you rename the `kubevirt-privileged` account into `kubevirt-virt-handler`, or something similar?